Merge master into desktop-ebook branch

macastejon opened the pull request:
## Summary

Initial commit for the book's contents

Commit:
Reorganizing tutorial for flow

Commit:
Updated tutorial to match updated code sample

Commit:
Adding reference to IDataView Type System post

Commit:
Consistent capitalization

schwastek committed:
Update work-with-data-in-asp-net-core-apps.md

Add syntax highlighting to console code blocks.

BillWagner committed:
remove extra "tuples" subdirectory (#14055)

Companion to dotnet/samples#1349

When I updated the tuples sample, I removed the extra subdirectory created by the framework templates.
This PR updates the code includes.

Commit:
Merge pull request #14052 from schwastek/patch-1

Update work-with-data-in-asp-net-core-apps.md

widjesh committed:
Correction in "Develop ASP.NET Core MVC apps" (#13812)

* Correction

* Compiler Errors removed

Commit:
Fixed flow

Commit:
Updated title

luisquintanilla committed:
Merge pull request #13920 from luisquintanilla/onnx-reorder

Reorganize ONNX Tutorial for better flow

luisquintanilla committed:
Merge pull request #14031 from luisquintanilla/issue/13732

Add reference to IDataView Type System post

luisquintanilla committed:
Merge pull request #14030 from luisquintanilla/issue/13788

Update Movie Recommendation Tutorial

mairaw committed:
remove broken link (#14054)

nxtn committed:
Replace Windows Vista tokens (#14050)

* Replace Windows Aero tokens

* Remove tlasharptla-aero-md.md

* Replace Windows Vista tokens

* Remove tlasharptla-longhorn-md.md

* Replace Windows Vista tokens

* Remove tlasharptla-longhorn2-md.md

* Apply suggestions from code review

Co-Authored-By: Ron Petrusha <ronpet@microsoft.com>

* Update ui-automation-security-overview.md

* Update technology-regions-overview.md

Commit:
Update reflection.md

Commit:
Changed links to Formatting Utility (#14071)

mairaw committed:
Update .NET Core telemetry (#14076)

* Update .NET Core telemetry

* Apply suggestions from docs review

Co-Authored-By: Ron Petrusha <ronpet@microsoft.com>

nxtn committed:
Replace .NET tokens (#14067)

* Replace Microsoft .NET tokens

* Remove tlasharptla-net-md.md

* Replace .NET tokens

* Remove tla2sharptla-net-md.md

* Apply suggestions from code review

Co-Authored-By: Maira Wenzel <mairaw@microsoft.com>

* Update globalization-for-wpf.md

pkulikov committed:
Moved and revised the await operator article (#14080)

* Moved the await article

* Revised the await article

* Minor edits

* Update docs/csharp/programming-guide/concepts/async/index.md

Co-Authored-By: Bill Wagner <wiwagn@microsoft.com>

* Updated method name

* Addressed feedback

kenyondc committed:
Migration of Legacy VeriSign Time Stamping Services (#14046)

See - https://knowledge.digicert.com/alerts/migration-of-legacy-verisign-and-symantec-time-stamping-services.html
As part of our rebranding initiative from the acquisition of Symantec in 2017, DigiCert has stopped future timestamp signatures from legacy Verisign timestamp services and facilitate all future timestamps via our consolidated DigiCert timestamping service.

Commit:
Attempted to clarify contravariance example (#14072)

* Attempted to clarify contravariance example

* Corrected syntax, added VB version

* Corrected language ids

Commit:
Clarify data loading

luisquintanilla committed:
Merge pull request #14090 from luisquintanilla/issue/13873

Clarify data loading language in ML.NET How-To's

Commit:
Modified breaking change template (#14004)

* Modified breaking change template

* Update doc-breaking-change.md

Commit:
Batch 2 (#14093)

nemrism committed:
Restore config file hierarchy and links (#14088)

* restore config file xml hierarchy

contributes to #2038

* standardize formatting

nxtn committed:
Replace .NET Framework attribute tokens (#14068)

* Replace .NET Framework attribute tokens

* Remove tlasharptla-netframewkattr-md.md

* Apply suggestions from code review

Co-Authored-By: Maira Wenzel <mairaw@microsoft.com>

* Update typeconverters-and-xaml.md

* Update type-converters-for-xaml-overview.md

nemrism committed:
[framework] Remove redundant path segments (part 5) (#14000)

* remove redundant path segments

/framework/data/transactions/*.md

* remove redundant path segments

/framework/data/adonet/dataset-datatable-dataview/*.md

* Update enumeration-classes-over-enum-types.md

* remove redundant path segments

/framework/data/adonet/index.md

* remove redundant path segments

/framework/misc/*.md

* fix link replacement errors

mairaw committed:
update link to use docs article (#14063)

ite-jankl committed:
Update link from obsolete to current event (#14081)

* Update link from obsolete to current event

The Form.Closing event is obsolete since .NET Framework version 2.0 as
per its documentation.
It has been replaced by the Form.FormClosing event, which we now link to
instead.

* Replace two more occurances of obsolete event

* Update EventArg reference for updated event

Co-Authored-By: Ron Petrusha <ronpet@microsoft.com>

nxtn committed:
Replace XML namespace tokens (#14107)

* Replace `http://schemas.openxmlformats.org/markup-compatibility/2006` tokens

* Remove tlasharptla-mcxmlnsv1-md.md

mairaw committed:
Update ms.topic (#14095)

Commit:
Batch 5 (#14118)

mairaw committed:
fix code formatting (#14096)

Commit:
Update exposing-dotnet-components-to-com.md (#14120)

Commit:
Added ASP.NET back to category list (#14116)

Commit:
Batch 3 (#14112)

cartermp committed:
UnionCaseName -> UnionCaseIdentifier (#14113)

fixes #14111

dasMulli committed:
Add missing muxer CLI arguments (#12545)

* Add missing muxer CLI arguments

* updates

Commit:
US 1583733 Add missing language IDs - 04 (#14115)

* Batch 4

* Fixes per review

Commit:
Update .NET Standard table for 2.1 (#14119)

* Update .NET Standard table for 2.1

* update ms.date

* Apply suggestions from code review

Co-Authored-By: Maira Wenzel <mairaw@microsoft.com>

* Add title & link

* Fix Android version based on Xamarin's feedback

* Update net-standard-table.md

* Update net-standard-table.md

nemrism committed:
[framework] Remove redundant path segments (part 6) (#14035)

* remove redundant path segments

/framework/data/adonet/ef/language-reference/*.md

* remove redundant path segments

/framework/data/adonet/ef/language-reference/*.md

* remove redundant path segments

/framework/data/adonet/ef/language-reference/*.md

* remove redundant path segments

/framework/data/adonet/ef/language-reference/*.md

* remove redundant path segments

/framework/data/adonet/ef/language-reference/*.md

nemrism committed:
[framework] Remove redundant path segments (part 7) (#14036)

* remove redundant path segments

/framework/data/adonet/ef/language-reference/*.md

* remove redundant path segments

/framework/data/adonet/ef/*.md

* remove redundant path segments

/framework/data/adonet/sql/linq/*.md

dnfclas commented:
[![CLA assistant check](https://cla.dotnetfoundation.org/pull/badge/not_signed)](https://cla.dotnetfoundation.org/dotnet/docs?pullRequest=14145) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/dotnet/docs?pullRequest=14145) before we can accept your contribution.<br/><br/>:x: macastejon [sign now](https://cla.dotnetfoundation.org/dotnet/docs?pullRequest=14145)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/dotnet/docs?pullRequest=14145) it.</sub>

poulad committed:
Update page size values (#14102)

Commit:
US 1583733 Add missing language IDs - 06 (#14154)

* Batch 6 - part 1

* Batch 6 - part 2

* delete extra spaces

pkulikov committed:
C# 8.0: enhancement of interpolated verbatim strings (#14138)

* C# 8.0: enhancement of interpolated verbatim strings

* Update ms.date

* Remove examples

Youssef1313 committed:
Fix language identifier (#14128)

* Fix language identifier

* Update name-name-is-not-declared.md

* ToInteger -> ChangeType

* Update docs/visual-basic/language-reference/error-messages/name-name-is-not-declared.md

Co-Authored-By: Next Turn <45985406+NextTurn@users.noreply.github.com>

BillWagner committed:
Update language reference to include `notnull` constraint (#14075)

* Update language reference to include `notnull` constraint

The `notnull` constaint has been added to C# 8.0
This PR updates the language reference and programming guide pages with that information.

* Apply suggestions from code review

Co-Authored-By: Petr Kulikov <petr.kulikov@gmail.com>

* respond to feedback + grammar check

This is ready for final review.

* Update docs/csharp/programming-guide/generics/constraints-on-type-parameters.md

Co-Authored-By: Tom Dykstra <tdykstra@microsoft.com>

* respond to feedback.

* fix build warning

* one more time with the build warning.

mairaw commented:
@macastejon this was not really new content from your branch so I manually merged master offline. You probably need a new PR.
